launcher/minecraft/mod/ModFolderModel.cpp

```diff
@@ -195,6 +213,10 @@ QVariant ModFolderModel::headerData(int section, [[maybe_unused]] Qt::Orientatio
                     return tr("The release type.");
                 case SizeColumn:
                     return tr("The size of the mod.");
+                case RequiredByColumn:
+                    return tr("Number of mods for what this is needed.");
+                case RequiresColumn:
+                    return tr("Number of mods that this requires.");
```

Comment:
I would prefer this:
```suggestion
                case RequiredByColumn:
                    return tr("For each mod, the number of other mods which depend on it.");
                case RequiresColumn:
                    return tr("For each mod, the number of other mods it depends on.");
```

Might be inconsistent but I want it to be easier to understand than time travel

Reply:
Also unresolved.